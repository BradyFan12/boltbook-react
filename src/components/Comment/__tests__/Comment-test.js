import React from 'react';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Comment } from '../';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

test('should render as expected', () => {
  const component = shallow(<Comment />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});