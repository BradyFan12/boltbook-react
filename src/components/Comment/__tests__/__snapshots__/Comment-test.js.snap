// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render as expected 1`] = `
<div
  className="comment"
>
  <div
    className="avatar"
  >
    <img
      alt=""
      src=""
    />
  </div>
  <input
    className="form-control"
    placeholder="Write a comment..."
    type="text"
  />
</div>
`;
